scss/_patterns_modal.scss

```diff
@@ -4,15 +4,15 @@
   .p-modal {
     align-items: center;
```

Comment:
margin-bottom on the child throws this vertical centering off. I would set it to 0.